pkg/cli/cli.go

```diff
@@ -32,323 +32,342 @@ import (
 )
 
 const (
-	noticeColor         = "\033[1;36m%s\033[0m"
-	runInProjectRootMsg = `For project-specific information, run this command in the root directory of a
-project.
-`
+	noticeColor = "\033[1;36m%s\033[0m"
 
 	projectVersionFlag = "project-version"
-	helpFlag           = "help"
 	pluginsFlag        = "plugins"
 )
 
+// equalStringSlice checks if two string slices are equal.
+func equalStringSlice(a, b []string) bool {
```

Comment:
Does order matter here?

Reply:
Yes, which plugins gets executed first is important.